netcdf/attributes.go

```diff
@@ -40,6 +45,22 @@ func (a Attr) Len() (n uint64, err error) {
 	return
 }
 
+// ValueString returns the value of the attribute value of type text.
+func (a Attr) ValueString() (val string, err error) {
```

Comment:
What's wrong with using Var's ReadBytes method and converting the resulting []byte to a string?